presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -309,9 +309,9 @@ public OperatorStats add(Iterable<OperatorStats> operators)
         long systemMemoryReservation = this.systemMemoryReservation.toBytes();
         Optional<BlockedReason> blockedReason = this.blockedReason;
 
-        Mergeable<?> base = null;
+        Mergeable<OperatorInfo> base = null;
         if (info instanceof Mergeable) {
-            base = (Mergeable<?>) info;
+            base = (Mergeable<OperatorInfo>) info;
```

Comment:
This is an unchecked cast. Change this to a helper method which can suppress the warning:
```java
Mergeable<OperatorInfo> base = getOptionalMergableInfo();